bitnami/nginx-ingress-controller/values.yaml

```diff
@@ -240,7 +240,7 @@ livenessProbe:
   enabled: true
   httpGet:
     path: /healthz
-    port: "{{ .Values.containerPorts.metrics }}"
+    port: 10254
```

Comment:
Better approach:
```suggestion
port: metrics
```

Reply:
It can't be used as `metrics=10254` is only used when `metrics.enabled=true`